modules/standard/Random.chpl

```diff
@@ -582,18 +582,9 @@ module Random {
         PCGRandomStreamPrivate_count = 1;
       }
 
-      //
-      // NOAKES: 2016/04/21
-      //
-      // syncs are currently implemented as bare classes and so require a delete to be reclaimed
-      //
-      // We do not want to update application code but have agreed to update the randomStream
-      // classes to reduce leak noise until the implementation can be revised
-      //
-
       pragma "no doc"
       proc ~RandomStream() {
```

Comment:
You are correct of course.  Applied.

Mike



On Aug 9, 2016, at 4:54 PM, Ben Harshbarger <notifications@github.com<mailto:notifications@github.com>> wrote:


In modules/standard/Random.chpl<https://github.com/chapel-lang/chapel/pull/4309#discussion_r74165569>:

> @@ -582,18 +582,9 @@ module Random {
>          PCGRandomStreamPrivate_count = 1;
>        }
>
> -      //
> -      // NOAKES: 2016/04/21
> -      //
> -      // syncs are currently implemented as bare classes and so require a delete to be reclaimed
> -      //
> -      // We do not want to update application code but have agreed to update the randomStream
> -      // classes to reduce leak noise until the implementation can be revised
> -      //
> -
>        pragma "no doc"
>        proc ~RandomStream() {


Could/should these empty destructors be removed?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/chapel-lang/chapel/pull/4309/files/579289b713603dc02bb18e1be47cdc8c92877849#r74165569>, or mute the thread<https://github.com/notifications/unsubscribe-auth/ACERplekRMqANkjbhhg_ohPyIo2diWPRks5qeRMngaJpZM4JgnUH>.